### PR TITLE
Modified Accuracy class and added docstrings

### PR DESCRIPTION
**Pull request by hassiahk**
- Modified Accuracy class to allow users to set the threshold value and to pass in logits as predictions instead of actual labels.
- Modified Precision and Recall class
- Updated Readme.md
- Added some docstrings

### CODE REVIEW

**Comment by hassiahk** [comment]
@enochkan There are different accuracy metrics like `Accuracy`, `BinaryAccuracy`, `CategoricalAccuracy`, `SparseCategoricalAccuracy` in `tf.keras.metrics` module but I actually merged everthing into one. How do you want to have these metrics?

**Comment by enochkan** [comment]
@hassiahk I actually don't mind having them all in one, just make sure you update the documentations.

**Comment by hassiahk** [comment]
Okay, I will see if it is better to have separate classes for these.
